src/scripts.js

```diff
@@ -7,30 +7,25 @@ import RecipeCollection from '../src/classes/RecipeCollection';
 const allRecipesSection = document.getElementById('allRecipes');
 const userName = document.getElementById('userGreeting');
 const recipeModal = document.getElementById('modal')
-// const favorite = document.getElementById('favoriteHeart')
 const recipeTags = document.getElementById('recipeTags');
+const favoriteRecipe = document.querySelector('.favorite-icon');
 let currentUser, recipeCollection, ingredients, recipe;
 
 window.addEventListener('load', onPageLoad);
-window.addEventListener('click', closeModal);
```

Comment:
Did you remove this because you were bug hunting? We need it to close the modal.